tests/test_sindri.py

```diff
@@ -0,0 +1,111 @@
+import os
+import time
+
+from src.sindri.sindri import Sindri
+
+
+# Load sample data.
+# This assumes the sindri-resources repo is cloned in the parent dir of this repo
+dir_name = os.path.dirname(os.path.abspath(__file__))
+sindri_resources_path = os.path.join(dir_name, "..", "..", "sindri-resources")
+assert os.path.exists(sindri_resources_path)
+
+circuit_dir = os.path.join(sindri_resources_path, "circuit_database", "noir", "not_equal")
+proof_input_file_path = os.path.join(circuit_dir, "Prover.toml")
+assert os.path.exists(circuit_dir)
+assert os.path.exists(proof_input_file_path)
+proof_input = ""
+with open(proof_input_file_path, "r") as f:
+    proof_input = f.read()
+
+
+# Create an instance of the Sindri SDK
+api_key = os.environ.get("SINDRI_API_KEY", "unset")
+api_url = os.environ.get("SINDRI_API_URL", "https://sindri.app/api")
```

Comment:
With API tests, I think you'd typically use something like [vcrpy](https://vcrpy.readthedocs.io/en/latest/) to record responses, then have separate test commands for recording vs playing back. I'll buy it being out of scope for this PR, but doing something like that and running tests on CI is probably the direction we want to head with this.

Reply:
I created a new issue for updating the unit tests with recorded responses: #23